tests/test_views.py

```diff
@@ -77,3 +77,17 @@ def test_post_list_rendering(self):
         self.assertContains(response, post)
         self.assertContains(response, past_post)
         self.assertNotContains(response, future_post)
+
+    def test_post_detail(self):
+        """Test display of post detail view based on post's existence in db."""
```

Comment:
This is a very vague description of something being checked without specifying what one behavior is expected.

Reply:
Ok, I tried to make this more detailed.